client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
We should make the case_id a little more robust since a student could add their own test cases or delete a test case or something and ruin the case_id. How about we just have the case_id be an internal thing that's a cryptographic hash of the question contents or something? Or we can leave the case_id alone and introduce a unique_id that's the cryptographic hash.